docs/notebooks/.ipynb_checkpoints/WMD_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,601 @@
+     "output_type": "stream",
+     "text": [
+      "distance = 1.3604\n"
+     ]
+    }
+   ],
+   "source": [
+    "sentence1 = 'Obama speaks to the media in Illinois'\n",
+    "sentence3 = 'Oranges are my favorite type of fruit'\n",
+    "sentence1 = sentence1.lower().split()\n",
+    "sentence3 = sentence3.lower().split()\n",
+    "sentence1 = [w for w in sentence1 if w not in stop_words]\n",
+    "sentence3 = [w for w in sentence3 if w not in stop_words]\n",
+    "\n",
+    "distance = model.wmdistance(sentence1, sentence3)\n",
+    "print 'distance = %.4f' % distance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Normalizing word2vec vectors\n",
+    "\n",
+    "When using the `wmdistance` method, it is beneficial to normalize the word2vec vectors first, so they all have length equal to 1. To do this, simply call `model.init_sims(replace=True)` and Gensim will take care of that for you.\n",
+    "\n",
+    "The reason it is beneficial to use normalized vectors is that word2vec is optimized for cosine distance (see [cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity)), which measures the angle between vectors, but WMD uses Euclidean distance. The Euclidean distance between two vectors might be large because their lengths differ, but the cosine distance is small because the angle between them is small; we can mitigate some of this by normalizing the vectors.\n",
+    "\n",
+    "Note that normalizing the vectors can take some time, especially if you have a large vocabulary and/or large vectors.\n",
+    "\n",
+    "Usage is illustrated in the example below. It just so happens that the vectors we have downloaded are already normalized, so it won't do any difference in this case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cell took 11.987278 seconds to run.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Normalizing word2vec vectors.\n",
+    "start = time()\n",
+    "\n",
+    "model.init_sims(replace=True)  # Normalizes the vectors in the word2vec class.\n",
+    "\n",
+    "distance = model.wmdistance(sentence1, sentence2)  # Compute WMD as normal.\n",
+    "\n",
+    "print 'Cell took %f seconds to run.' %(time() - start)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Similarity queries\n",
+    "\n",
+    "You can use WMD to get the most similar documents to a query, using the `WmdSimilarity` class. Its interface is similar to what is described in the [Similarity Queries](https://radimrehurek.com/gensim/tut3.html) Gensim tutorial.\n",
+    "\n",
+    "> **Important note:**\n",
+    ">\n",
+    "> WMD is a measure of *distance*. The similarities in `WmdSimilarity` are simply the *negative distance*. Be careful not to confuse distances and similarities.\n",
```

Comment:
Be more explicit " Similar objects = high similarity, low distance. Different objects = low similarity, high distance."